src/store/index.js

```diff
@@ -0,0 +1,41 @@
+import { createStore } from 'vuex'
+
+export default createStore({
+  state() {
+    return {
+        // Array of selected images. Image objects that get selected get pushed here
+      selectedImages: []
+    }
+  },
+  mutations: {
+    // This mutation changes the state of selectedImages
+    // It should be dispatched when an image is selected/unselected
+    toggleImageSelection(state, image) {
```

Comment:
I'd like to follow ptr standards here for the store https://github.com/LCOGT/ptr_ui/blob/4aa6e8b4003241996c42d40b98d5367bc39b4d64/src/store/modules/project_params.js#L136C7-L136C16
where mutators and getters names are the same as the state they change or get. The actions can have more identifying names, but for scalability it helps for understanding which getters and mutators affect which state. But also ask Matt what he thinks this is just my opinion.

Reply:
I'd say in this case it's a little different than PTR - all of those mutators you linked to are simply setting a value to what's being passed in. If we were doing that here I'd absolutely agree. I'd say that toggling the image selection _is_ the mutation that's being performed here, so IMO the name is quite descriptive.

Reply:
do you have a suggestion?

Reply:
selected_images_toggle_selection?